rust/cardano-chain-follower/src/turbo_downloader/mod.rs

```diff
@@ -680,9 +702,20 @@ impl std::io::Read for ParallelDownloadProcessor {
 ///
 /// This exists because the `Probe` call made by Mithril is Async, and this makes
 /// interfacing to that easier.
-async fn get_content_length_async(url: &str) -> anyhow::Result<usize> {
+async fn get_content_length_async(url: &str, chain: Network) -> anyhow::Result<usize> {
     let url = url.to_owned();
-    match tokio::task::spawn_blocking(move || get_content_length(&url)).await {
+    match tokio::task::spawn_blocking(move || {
+        stats::start_thread(
```

Comment:
This is just a async - sync bridge, not a task or worker, so doesn't need stats.